test/e2e/manifests/datadog-agent-resources.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: datadoghq.com/v2alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog-resources
+  namespace: e2e-operator
+  labels:
+    agent.datadoghq.com/e2e-test: datadog-agent-minimum
```

```suggestion
    agent.datadoghq.com/e2e-test: datadog-agent-dap
```
to differentiate between other DDA yamls